k4GeneratorsConfig/src/pythiaLHERunner.cxx

```diff
@@ -1,89 +1,120 @@
-// main07.cc is a part of the PYTHIA event generator.
```

Comment:
This comes from the Pythia tutorial path and the code in this repo should not have Pythia's copyrights.